File: app/src/main/java/com/organizobra_mobile/ui/projects/ProjectsViewModel.java
package com.organizobra_mobile.ui.projects;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class ProjectsViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public ProjectsViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is home fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/organizobra_mobile/ui/projects/ProjectsViewModel.kt
package com.organizobra_mobile.ui.projects

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class ProjectsViewModel : ViewModel() {
    private val mText = MutableLiveData<String>()

    init {
        mText.value = "This is home fragment"
    }

    val text: LiveData<String>
        get() = mText
}